diff --git a/templates/nsis/multiUser.nsh b/templates/nsis/multiUser.nsh
index c345dee..675f0d7 100644
--- a/templates/nsis/multiUser.nsh
+++ b/templates/nsis/multiUser.nsh
@@ -2,0 +3 @@
+!include WinVer.nsh
@@ -31,9 +32,11 @@ Var installMode
-      System::Store S
-      # Win7 has a per-user programfiles known folder and this can be a non-default location
-      System::Call 'SHELL32::SHGetKnownFolderPath(g "${FOLDERID_UserProgramFiles}", i ${KF_FLAG_CREATE}, p 0, *p .r2)i.r1'
-      ${If} $1 == 0
-        System::Call '*$2(&w${NSIS_MAX_STRLEN} .s)'
-        StrCpy $0 $1
-        System::Call 'OLE32::CoTaskMemFree(p r2)'
-      ${endif}
-      System::Store L
+      ${IfNot} ${AtLeastWin8}
+        System::Store S
+        # Win7 has a per-user programfiles known folder and this can be a non-default location
+        System::Call 'SHELL32::SHGetKnownFolderPath(g "${FOLDERID_UserProgramFiles}", i ${KF_FLAG_CREATE}, p 0, *p .r2)i.r1'
+        ${If} $1 == 0
+          System::Call '*$2(&w${NSIS_MAX_STRLEN} .s)'
+          StrCpy $0 $1
+          System::Call 'OLE32::CoTaskMemFree(p r2)'
+        ${endif}
+        System::Store L
+      ${EndIf}
